make-release.js problems?

When trying to use `make-release.js`, with node 16.16, I am getting:

```
> uhk-agent@2.0.2 convert-user-config-to-bin
> node -r ts-node/register ./packages/usb/user-config-json-to-bin.ts "/opt/firmware/scripts/uhk-firmware-9.1.4/devices/uhk60-right/config.bin"

TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /opt/firmware/lib/agent/packages/usb/user-config-json-to-bin.ts
    at new NodeError (node:internal/errors:372:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:76:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:118:38)
    at defaultLoad (node:internal/modules/esm/load:21:20)
    at ESMLoader.load (node:internal/modules/esm/loader:407:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
    at async Promise.all (index 0) {
  code: 'ERR_UNKNOWN_FILE_EXTENSION'
}
/opt/firmware/scripts/node_modules/shelljs/src/common.js:401
      if (config.fatal) throw e;
                        ^

Error: exec: TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /opt/firmware/lib/agent/packages/usb/user-config-json-to-bin.ts
    at new NodeError (node:internal/errors:372:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:76:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:118:38)
    at defaultLoad (node:internal/modules/esm/load:21:20)
    at ESMLoader.load (node:internal/modules/esm/loader:407:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:326:22)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:345:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:304:34)
    at async Promise.all (index 0) {
  code: 'ERR_UNKNOWN_FILE_EXTENSION'
}

    at Object.error (/opt/firmware/scripts/node_modules/shelljs/src/common.js:110:27)
    at execSync (/opt/firmware/scripts/node_modules/shelljs/src/exec.js:102:12)
    at _exec (/opt/firmware/scripts/node_modules/shelljs/src/exec.js:205:12)
    at /opt/firmware/scripts/node_modules/shelljs/src/common.js:335:23
    at Object.<anonymous> (/opt/firmware/scripts/make-release.js:55:5)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
```

When trying to use `make-release.js, with node 12.22.12 (as firmware/README.md instructs), but Agent built with 16.16, I am getting:

edit:
I.e. I am trying to:
```
sudo n 16
cd lib/agent
npm ci && npm run build
sudo n 12
cd ../../scripts
npm ci 
./make-release.js
```

```
> uhk-agent@2.0.2 postinstall /opt/firmware/lib/agent
> lerna bootstrap

/opt/firmware/lib/agent/node_modules/@lerna/package-graph/index.js:79
            if (depNode?.version) {
                        ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/opt/firmware/lib/agent/node_modules/@lerna/query-graph/query-graph.js:3:26)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! uhk-agent@2.0.2 postinstall: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the uhk-agent@2.0.2 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/me/.npm/_logs/2022-11-20T15_04_38_224Z-debug.log
```

Any ideas what is wrong? :-/

(@ert78gb I assume you are most knowledgeable in this area?)

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ Going forward, it's easier to flash the firmware of your choice by using the dow
 
 1. Install the ARM cross-compiler, cross-assembler and stdlib implementation. Eg. on Arch Linux the packages `arm-none-eabi-binutils`, `arm-none-eabi-gcc`, `arm-none-eabi-newlib`.
 
-2. Install Node.js v12. If you have a later version, editing the version requirement in `lib/agent/package.json` *might* work.
+2. Install Node.js. You find the expected Node.js version in `lib/agent/.nvmrc` file. Use your OS package manager to install it. [Check the NodeJS site for more info.](https://nodejs.org/en/download/package-manager/ "Installing Node.js via package manager") Mac OS users can simply `brew install node` to get both. Should you need multiple Node.js versions on the same computer, use Node Version Manager for [Mac/Linux](https://github.com/creationix/nvm) or for [Windows](https://github.com/coreybutler/nvm-windows)
 
 3. Build UHK Agent. `cd lib/agent && npm ci && npm run build`.
 

